package/AUTHORS

```diff
@@ -155,7 +155,7 @@ Chronological list of authors
 2021
   - Aditya Kamath
   - Leonardo Barneschi
-  
+  - Ahmad Nasir
 External code
```

Comment:
Please make sure that you keep the empty line between the author list and the "External code" heading.